google/cloud/forseti/services/inventory/base/resources.py

```diff
@@ -2002,6 +2006,25 @@ def iter(self):
                 # API client doesn't support this resource, ignore.
                 LOGGER.debug(e)
 
+class GsuiteGroupSettingsIterator(ResourceIterator):
+    """The Resource iterator implementation for Gsuite Group Settings"""
+
+    def iter(self):
+        """Resource iterator.
+
+        Yields:
+            Resource: GsuiteGroupSettings created
+        """
+        gsuite = self.client
+        if self.resource.has_directory_resource_id():
+            try:
+                for data in gsuite.iter_gsuite_group_settings(
+                        self.resource['owner']['directoryCustomerId']):
```

Comment:
ensure that this ID is relevant to our settings api

Reply:
This iterator needs to be a child of the GsuiteGroupIterator. The group is the parent resource for the settings.